pkg/daemon/update.go

```diff
@@ -1102,8 +1102,9 @@ func getSupportedExtensions() map[string][]string {
 	// These are RHCOS supported extensions.
 	// Each extension keeps a list of packages required to get enabled on host.
 	return map[string][]string{
-		"usbguard":     {"usbguard"},
-		"kernel-devel": {"kernel-devel", "kernel-headers"},
+		"usbguard":             {"usbguard"},
+		"kernel-devel":         {"kernel-devel", "kernel-headers"},
+		"sandboxed-containers": {"ipxe-roms-qemu", "kata-containers", "libpmem", "pixman", "qemu-kiwi", "qemu-kvm-common", "seabios-bin", "seavgabios-bin", "sgabios-bin"},
```

Comment:
You can specify here top level packages that you need to enable the functionality. You don't need to specify those package which will for sure gets pulled in as dependency, rpm-ostree takes care of resolving and downloading deps.

For example, on my regular Fedora system I want to install qemu. So, I will just do `dnf install qemu` and qemu packagers make sure that dnf pulls in deps they need for qemu to work.

Reply:
Oh, that's lovely to know. So, we just need to ensure we have the correct repos enabled on RHCOS side.

Thanks a lot for the review, @sinnykumari!

Reply:
Ah, @sinnykumari, down below in this PR, we do the check about the expected and installed packages.
Would there make sense to have all the deps listed? Or are we good to go just with`kata-containers`?

Reply:
If `kata-containers` package ensures that it will install deps, then yes just verifying for kata-containers should be enough.

Reply:
> Oh, that's lovely to know. So, we just need to ensure we have the correct repos enabled on RHCOS side.

yes, as per https://github.com/openshift/enhancements/pull/677/files#diff-3baabdf22a2dfc2542b74496b417654de8a95693f9d1bb09605c066afb790ed1R471 I think enabling required repos and specifying just `kata-containers` for sandboxed-containers` extension in https://github.com/openshift/os/blob/master/extensions.yaml should be enough.